src/main.js

```diff
@@ -138,3 +142,63 @@ const killAllRunningProcesses = () => {
     console.error('Got error when trying to kill children', err);
   }
 };
+
+const manageApplicationLocation = () => {
+  if (canApplicationBeMoved()) {
+    showMoveToApplicationsFolderDialog();
```

Comment:
I feel like the name of this function is a little misleading, since it doesn't just show a dialog, but it processes the response.

Also, this is 100% subjective, but I'm a fan of less abstraction than this =) my personal philosophy is that while putting every "thing" in its own function makes code feel very modular and clean, it also makes it a bit harder to follow; the reader has to jump to another part in the file, and remember the callstack to understand the full context.

Ultimately it's your call (I try not to be too heavy-handed with things that are reasonably subjective), but I would just inline the `showMoveToApplicationsFolderDialog` function here. If you'd prefer to keep it separate, please just give it a more accurate name like `moveApplicationIfUserChooses`

Reply:
I agree with you :) My initial reasoning was that by doing this, it would be easier if in the future tests would be added for this feature and both parts of the function could be tested individually, but as you said, it is a bit misleading that the `showMoveApplicationsFolderDialog` is named that way when it actually has a "side effect" besides showing the dialog. I like your premise about abstraction, as i think it also facilitates the interpretation of the codebase by the reader, so i will change it to have the function inline :)